src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -1486,4 +1487,32 @@ void createConsumerWithInvalidFormat(VertxTestContext context) throws Interrupte
         create.get(TEST_TIMEOUT, TimeUnit.SECONDS);
         context.completeNow();
     }
+
+    @Test
+    void createConsumerNameIsNotSetAndBridgeIdIsSet(VertxTestContext context) throws InterruptedException, ExecutionException, TimeoutException {
```

Comment:
not sure why this test is in this file and not in `ConsumerGeneratedNameTest` with the other ones.

Reply:
the test that i added here is because of shared bridge. All the test cases in the ConsumerGeneratedNameTest has null bridge id and other ones has bridge id my-bridge

Reply:
Yes right.